examples/source/interactivity-dynamic-content/Faux_Lightbox_Overlay.html

```diff
@@ -0,0 +1,96 @@
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <style amp4email-boilerplate>body{visibility:hidden}</style>
+  <!--
+    ## Styles
+
+    Use CSS to define the styles for a background overlay and individual overlays displayed when a lightbox is open.
+  -->
+  <style amp-custom>
+    #lightbox-background {
+      position: absolute;
+      background-color: rgba(255, 255, 255, 0.9);
+      left: 0;
+      right: 0;
+      top: 0;
+      bottom: 0;
+    }
+
+    #lightbox-background.open {
+      display: block;
+    }
+
+    .lightbox-container {
+      position: relative;
+    }
+
+    .lightbox-overlay {
+      position: absolute;
+      top: 0;
+      z-index: 1;
+    }
+  </style>
+</head>
+<!-- ## Implementation -->
+<!-- -->
+<body>
+  <!--
+    Wrap the initially visible content as well as overlay content into an element with the `lightbox-container` class.
+
+    We use the `overlays` object to hold the state of all overlays on the page. Each overlay has a separate property inside this object it binds to, in this case `overlays.o1`.
+
+    Pressing the "Show overlay" button changes the state, displaying both the overlay and the background on the whole page.
+  -->
+  <div class="lightbox-container">
+    <p style="padding-top: 100px">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris non porta metus, eget lobortis metus. Duis vel arcu quis leo auctor vulputate commodo ac magna. Curabitur suscipit sit amet nisl quis pulvinar. Donec at purus non nisl molestie fringilla. Quisque consequat ligula ut ligula pellentesque mollis. Phasellus a rhoncus risus. Curabitur molestie lorem metus, placerat lobortis est molestie finibus. Pellentesque viverra nulla in turpis ultrices, ac placerat dolor semper. Praesent rutrum ullamcorper mi, non suscipit magna finibus ut. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed nulla lectus, lacinia vitae nisi vitae, laoreet cursus magna. Praesent lacus est, vestibulum sed urna non, tristique fermentum felis. Nunc posuere porta nunc non auctor. Praesent fringilla rutrum iaculis. Duis euismod facilisis ligula, vel auctor mi pulvinar vitae.</p>
+    <button on="tap:AMP.setState({ overlays: { o1: true } })">Show overlay</button>
+
+    <div class="lightbox-overlay" hidden [hidden]="!overlays.o1">
+      This is an overlay.
+    </div>
+  </div>
+
+  <!--
+    A single page can contain multiple overlays.
+
+    In this case, tapping on the text itself opens the second overlay. The state property used by this overlay is different than the one above: `overlays.o2`.
+  -->
+  <div class="lightbox-container">
```

Comment:
I find it a bit weird to have multiple Lightbox containers.

Reply:
Addressed above, it's used to "anchor" the `position: absolute` for the content. I renamed it overlay-container, but not sure if that makes more sense.